tests/threshold/threshold-config-rate-filter-alert-flow/test.yaml

```diff
@@ -0,0 +1,25 @@
+pcap: ../../http-all-headers/input.pcap
+
+requires:
+  min-version: 8
+
+args:
+- --set threshold-file=${TEST_DIR}/threshold.config
+- --simulate-ips
+
+retry: 3
+
+checks:
+  - filter:
+      count: 19
+      match:
+        event_type: alert
+        alert.signature_id: 1000001
+  - filter:
+      count: 1
+      match:
+        event_type: drop
```

Comment:
Would like a comment or README stating something like

The first packet (syn without payload) triggers the rule with its original action being a drop
Then, all 19 "empty" packets (Wireshark filter `tcp && !(len(tcp.payload) > 0)`) (including the first one) will trigger the new action defined by the rate filter so alert (and no drop)

Reply:
Not seeing the point. Test is simple enough.